content/pages/faq.md

```diff
@@ -19,13 +19,36 @@ Holding events which are only open to women helps some women deal with
 [stereotype threat](http://geekfeminism.wikia.com/wiki/Stereotype_threat
 "Geek Feminism Wiki: Stereotype Threat").
 
-An example of an event that's women-only is our mentoring program, where we
-match up junior female students with more senior female students to talk about
-their experiences and support them throughout their academic career.  Women can
-obviously seek out male mentors as well, but they don't quite share the same
-experiences so the impact of the mentoring is not the same. There are also
-events that are traditionally very competitive (such as hackathons) that women
-feel more comfortable coming out to if they're women-only.
+There are a few categories of events that the Undergraduate Committee
+generally chooses to keep women only. These include: workshops,
+networking, and menotring or discussion circle-style events.
+
+Women are often excluded from these kinds of events when held by other
+organizations, sometimes because of unconcious biases playing a part 
+in the admissions process, or because some women do not feel comfortable
+at this kind of event when they know the majority of attendants will
+be men (see our link on stereotype threat above). We find 
+that holding these events as women-only is a way to alleviate this issue a
+ little for women in our community.
+
+To the criticism that we offer opportunities that are not available to men
```

Comment:
Maybe something like "To address the criticism that ..."? idk I had a hard time reading this sentence at first